src/test/java/com/github/anba/es6draft/test262/Test262Info.java

```diff
@@ -400,12 +400,12 @@ public void setEs7id(String es7id) {
             this.es7id = es7id;
         }
 
-        public String getId() {
-            return id;
+        public String getEsid() {
+            return esid;
         }
 
-        public void setId(String id) {
-            this.id = id;
+        public void setId(String esid) {
```

Comment:
my bad.

Also, sorry for the noise with the whitespace, caused by default settings on the editor. I can revert these if you want.